code/modules/modular_computers/hardware/hard_drive.dm

```diff
@@ -178,16 +178,21 @@
 	store_file(new /datum/computer_file/program/crew_manifest(src)) // SKYRAT EDIT ADD
 
 // For borg integrated tablets. No downloader.
-/obj/item/computer_hardware/hard_drive/small/ai/install_default_programs()
+/obj/item/computer_hardware/hard_drive/small/integrated/install_default_programs() // SKYRAT EDIT ADDITION - cyborg PDA messenger
 	var/datum/computer_file/program/messenger/messenger = new(src)
 	messenger.is_silicon = TRUE
 	store_file(messenger)
 
-/obj/item/computer_hardware/hard_drive/small/robot/install_default_programs()
+/obj/item/computer_hardware/hard_drive/small/integrated/borg/install_default_programs() // SKYRAT EDIT ADDITION - cyborg PDA messenger
 	store_file(new /datum/computer_file/program/computerconfig(src)) // Computer configuration utility, allows hardware control and displays more info than status bar
 	store_file(new /datum/computer_file/program/filemanager(src)) // File manager, allows text editor functions and basic file manipulation.
 	store_file(new /datum/computer_file/program/robotact(src))
 	store_file(new /datum/computer_file/program/crew_manifest(src)) // SKYRAT EDIT ADDITION - Manifests for cyborgs
+	/* SKYRAT EDIT ADDITION Start- Returns Messenger to borgs */
+	var/datum/computer_file/program/messenger/messenger = new(src) 
+	messenger.is_silicon = TRUE
+	store_file(messenger)
+	/* SKYRAT EDIT ADDITION End- Returns Messenger to borgs */
```

Comment:
TL;DR this is the _only_ change needed to get your messenger app back, this PR doesnt need any others

Reply:
Put this in a new modular folder